### DISCUSSION TITLE
Pager control

### DISCUSSION THREAD

**Discussion by PatrickKing** [ohmyzsh/ohmyzsh#10487] [Help] [answered]
What does OMZ do to the shell's pager, exactly?

Previously, many commands seemed to make their own decision about whether to use a pager or not. (e.g. `git diff` would always use a pager, but `git branch` would not). But with OMZ loaded, everything uses the pager it configures (PAGER is set to 'less'). But unsetting the pager variable or setting it to "" does not restore the old functionality, where some commands were paged and some were not. Instead, no commands use the pager.

How can I configure OMZ to use pagers (or perhaps not configure the pager) as before?
Thanks!

**Comment by PatrickKing** [comment, marked as answer]
I was able to get back the old behaviour (mostly) by setting PAGER after loading OMZ like so:

`export PAGER="less -F -X"`

I would guess that git uses default pager settings like this, but I don't know what that might be.

Edited: you might consider making this the default pager command, it might be more intuitive to most users, to judge by the large number of questions about this over the years.

**Reply by kyle-elyk-neo** [reply]
the -X flag was not one that I'd considered here, sneaky one

**Reply by FernandoBasso** [reply]
I believe it is worth mentioning that this also helped me with Bash (through Homebrew, not the old one that comes installed by default) on macOS.
